client/asset/btc/rpcclient.go

```diff
@@ -495,11 +495,35 @@ func (wc *rpcClient) listUnspent() ([]*ListUnspentResult, error) {
 	return unspents, wc.call(methodListUnspent, anylist{uint8(0)}, &unspents)
 }
 
+// lockAllUnspent unlocks all locked outputs for spending again. If unlock is
+// true btc will return 'true' if successful. If unlock is false btc will also
+// return 'true' but do nothing so a warning is issued below.
+//
+// See Also: btc_test.go: RawRequest case: methodLockUnspent
+func (wc *rpcClient) lockAllUnspent(unlock bool) error {
```

Comment:
Why accept an `unlock` argument at all? Also, why name the method `lock` when it **un**locks?